lib/rules/web/admin_console/4.5/browser_opts.yml

```diff
@@ -21,6 +21,16 @@
             scroll_by = browser.element(css: ".navbar.navbar-pf-vertical").height
             scroll_by += browser.element(css: ".project-bar").height
             browser.execute_script("window.scrollBy(0,-#{scroll_by})")
+          when /pf-c-form__actions/
+            logger.info "Scrolling from action button out of view"
+            scroll_by = browser.element(css: ".pf-c-form__actions").height
+            scroll_by += browser.element(css: ".ocs-radio-group-field").height
+            browser.execute_script("window.scrollBy(0,-#{scroll_by})")
```

Comment:
I didn't see this in the log but is it a header or a footer? If it is a footer then scrolling down makes more sense.